src/Video.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import './Video.css'
+
+const Video = ({video}) => {
+  console.log("video", video)
```

Comment:
Lingering `console.log`, may you missed it? Just pointing it out so you can delete it if you no longer need it.